### PR TITLE
Docs two point oh

### PR DESCRIPTION
**Pull request by soniktrooth**
stubbed out version based on scratch space doc

### CODE REVIEW

**Comment by soniktrooth** [comment]
Probs don't merge because this will replace the 'latest' RTD build with this.

File: docs/twig.md [hunk 1]
```diff
@@ -1,17 +0,0 @@
...
-This documents some of the fancy things you can do with Twig.
-
-## Extending Filters
-
-Add a `filters.metadata` file in the src directory....
-
-``` md
----
-slug: kalastatic-twig-filters/lib/filters/slug.js
----
```
Comment (RobLoach) [inline docs/twig.md]: Whole bunch of stuff is removed here? Intentional?

Comment (soniktrooth) [inline docs/twig.md]: Yeah, all the stuff that's getting removed is the 1.0 wiki stuff. I probably didn't do it well. Maybe we can cherry pick the 1.0 wiki stuff into the 1.0 tag and then remove the commit that adds them here.

Comment (soniktrooth) [inline docs/twig.md]: git is hard

**Pull request merged**